### DISCUSSION TITLE
Stop this sprite block!

### DISCUSSION THREAD

**Discussion by TheFirstOrangeScratcher7777** [ScratchAddons/ScratchAddons#5854] [Ideas] [answered]
I always when want to stop this sprite i use 2 or more bloks, I want to use 1 only, can be like in stop all block stop this sprite block too?
Oh, and, my name in Scratch is the same that here. 😁

**Comment by CST1229** [comment]
Why do you need to use only 1 block? (Adding new blocks that affect the project also wouldn't work for people not using SA.)

**Reply by TheFirstOrangeScratcher7777** [reply]
because in my games that really sometimes do bugs, then, i think that helps me and someone else,

**Comment by BroJac5246** [comment]
This is too complicated for a specific niche issue and is not worth it.

**Comment by DNin01** [comment]
So you want to combine these two blocks into one?
```
stop other scripts in sprite
stop this script
```

**Reply by TheFirstOrangeScratcher7777** [reply]
Me? So, sorry, but I wanna 1 block. :| `stop this sprite`

**Comment by Samq64** [comment, marked as answer]
Adding new blocks has been rejected in the past because it wouldn't be compatible without Scratch Addons. The debugger blocks are an exception because they're just custom blocks that don't do anything in normal Scratch and aren't meant to be used in finished projects. Also, using a `stop [other scripts in sprite v]` block without putting any blocks below it would do the same thing as a `stop this sprite` block.

**Reply by TheFirstOrangeScratcher7777** [reply]
Ok, now I understand why, fine. :)
